### DISCUSSION TITLE
Filter to only show files that are newer on one side?

### DISCUSSION THREAD

**Discussion by fraserbooth** [WinMerge/winmerge#926] [Q&A] [answered]
Thank you for the amazing software you make available :)

I've not found it straightforward to search the documentation, but I don't seem to be able to find if there is a way to filter out files that are only newer on one side. I would like to be able to merge two sets of folders, but only take "new" items from one side, without over writing files that have been update on the other side. Is there a plugin or filter that would allow me to do this?

**Comment by sdottaka** [comment, marked as answer]
Unfortunately, WinMerge does not have such a feature or plugin.

**Reply by fraserbooth** [reply]
Thank you for letting me know. I wonder if others would find this a useful feature?
